core/settings.py

```
@@ -15,6 +15,21 @@
 from pathlib import Path
 from dotenv import load_dotenv
 
+
+def env_bool(name, default=False):
```

Ruff (0.15.14)

[warning] Boolean default positional argument in function definition

(FBT002)